zerver/lib/actions.py

```diff
@@ -1142,11 +1216,12 @@ def set_stream_color(user_profile, stream_name, color=None):
     return color
 
 def get_subscribers_to_streams(streams):
+    # type: (Iterable[Stream]) -> Dict[UserProfile, List[Stream]]
```

Comment:
Can you add a TODO for cleaning this function up?  In general we've been avoiding having user objects be dict keys...